hack/lib/util/misc.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# This library holds miscellaneous utility functions. If there begin to be groups of functions in this
+# file that share intent or are thematically similar, they should be split into their own files.
+
+# os::util::describe_return_code describes an exit code
+#
+# Globals:
+#  - START_TIME
```

Comment:
By setting the value in the `'install'` function we ensure that this value is always present. This continues using the global but this is much less intensive than determining how to feed the correct value as an arg into the trap handler. I will implement this alternate approach, unless you see pitfalls to it.